pages/installation/coolify.mdx

```diff
@@ -87,7 +125,7 @@ services:
     container_name: postiz-redis
     restart: always
     ports:
-      - 6379:6379
+      - "7963:6379"
```

Comment:
And why change these?

Reply:
I think the ports are used by other instances, i will try with your defaults